bimmer_connected/cli.py

```diff
@@ -68,9 +65,8 @@ def get_status(args) -> None:
 
 def fingerprint(args) -> None:
     """Save the vehicle fingerprint."""
-    time_str = time.strftime("%Y-%m-%d_%H-%M-%S")
-    time_dir = os.path.join(FINGERPRINT_DIR, time_str)
-    os.makedirs(time_dir)
+    time_dir = Path.cwd() / 'vehicle_fingerprint' / time.strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
Should we maybe rather use `Path.home()` instead of `cwd()`? This would always be the user directory on Linux (`/home/username`) or Windows (`C:\Users\username`) and is available since Python 3.5.
Every user should have write permission to his own folder and also many cross-platform applcations use `C:\Users\username\.someapp` in Windows as well.

So maybe set `time_dir` like so:
```
Path.joinpath(Path.home(), '.bimmer_connected', 'vehicle_fingerprint', time.strftime("%Y-%m-%d_%H-%M-%S"))
```